script/script.js

```diff
@@ -0,0 +1,177 @@
+function displaySignUp() {
+	document.getElementById("signup").style.display="block";
+}
+
+function displayLogIn() {
+	document.getElementById("login").style.display="block";
+}
+
+function closeDialog() {
```

Comment:
While closing the dialog also remove any dom messages you would have added there.
If there is a message saying "incorrect credentials" just before closing the dialog, the same message should not reappear when the user reopens it.